### PR TITLE
Improvements in CI files

### PR DESCRIPTION
**Pull request by ibc**
This PR comes from PR #1280 which will be closed until intended changs are tested in a separate mediasoup fork. So just cosmetic CI changes here.

### CODE REVIEW

File: .github/workflows/mediasoup-node.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@ jobs:
   ci:
     strategy:
       matrix:
+        fail-fast: true
```
Comment (nazar-pc) [inline .github/workflows/mediasoup-node.yaml]: This should have been at the level higher here, but also it defaults to `true`, so I don't think it is really a necessary change: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast

Comment (ibc) [reply .github/workflows/mediasoup-node.yaml]: Which could explain why all those CI jobs are not even being runned XDDD. Thanks, fixing.

Comment (ibc) [reply .github/workflows/mediasoup-node.yaml]: OMG it's not being fixed... why this??

CleanShot-2023-12-29-at-10 33 15@2x

Comment (ibc) [reply .github/workflows/mediasoup-node.yaml]: ok typo introduced:

CleanShot-2023-12-29-at-10 34 31@2x

**Comment by ibc** [comment]
Ok, merging. Easy one.

**Pull request merged**